product_security/red_hat_responsibilities/access_control.adoc

```diff
@@ -0,0 +1,105 @@
+[[security-access-control]]
+= Access Control
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-authentication]]
+== Authentication
+
+Authentication for the cluster is configured as part of the on-boarding process.
+{product-title} includes a built-in OAuth server. The OAuth server
+generates two kinds of tokens:
+
+* _Access tokens_ are longer-lived tokens that grant access to the API.
+* _Authorize codes_ are short-lived tokens; their only use is to be exchanged for
+an access token.
+
+You can integrate your cluster with various identity providers at installation
+time, or later by opening a support case.
+
+Supported identity providers include:
+
+* Keystone
+* Google Auth
+* GitHub OAuth
+* OpenID Connect
+* LDAP
+
+[[security-encryption-and-key-management]]
+== Encryption and Key Management
+
+By default, a new internal public key infrastructure (PKI) is created for each
+deployment of {product-title}. The internal PKI uses 2048-bit RSA keys and
+SHA-256 signatures. The platform PKI is dedicated to managing the platform
+components’ certificates. This includes: the master API, controllers, and nodes.
+Also, by default, most of the infrastructure components deployed at install time
+are signed by this CA; for example: metrics, logging, and the default router
+wildcard certificate. By default, the PKI CA related files are stored on all
+OpenShift masters in the /etc/origin/master directory. The etcd cluster has its
+own PKI.
+
+Encryption keys for persistent volumes are managed by the Key Management service
+of the underlying infrastructure solution.
+
+[[security-authorization]]
+== Authorization
```

Comment:
Can we add this text right below the Authorization heading:

Authorization is managed using:
Rules : Sets of permitted verbs on a set of objects. For example, whether something can create pods.
Roles : Collections of rules. Users and groups can be associated with, or bound to, multiple roles at the same time.
Bindings: Associations between users and/or groups with a role.

OpenShift Dedicated evaluates authorizations using the following steps:

1. The identity and the project-scoped action is used to find all bindings that apply to the user or their groups.
2. Bindings are used to locate all the roles that apply.
3. Roles are used to find all the rules that apply.
4. The action is checked against each rule to find a match.
5. If no matching rule is found, the action is then denied by default.

Reply:
Is it okay if I omit the text and just move the link up to the content in xref:../../architecture/additional_concepts/authorization.adoc#architecture-additional-concepts-authorization[Learn
how authorization is managed] instead since that text is pulled from there? I don't want to duplicate content in our doc set.